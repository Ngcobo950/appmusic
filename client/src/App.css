.App {
  text-align: center;
}




body{
  background-image: url("apple 4.jpg");
 background-repeat: no-repeat, repeat;
 background-size: cover;
}

ul{
  text-align: center;

}

li{
    display: inline-block;
    padding: 10px 20px;
    font-weight: bold;
    text-align: center;
    

}

li:hover{
  color: white;
  background-color: grey;
  box-shadow: 0 0 10px #2196f3;
  text-align: center;
}

.head{
  text-align: center;
  border: 2px solid white;
  border-radius: 8px;
  border: 1px solid;
  padding: 10px;
  box-shadow: 5px 10px #888888;
} 



