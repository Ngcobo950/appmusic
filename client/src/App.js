import React from 'react';

import './App.css';
import { HashRouter as Router, Switch, Route, Link } from "react-router-dom";
import 'font-awesome/css/font-awesome.min.css';


import Home from './components/Home'
import Music from './components/Music';
import Books from './components/Books';
import Favorite from './components/Favorite';

//component that will render home, music and books and return them as a path
class App extends React.Component {
  cons
  render(){
    return (
      <Router >
        <Home/>
        <div>
          <nav>
            <ul>
              <li>
                <Link to={'/'}>Home</Link>
              </li>
              <li>
                <Link to={'/Music'}>Music</Link>
              </li>
              <li>
                <Link to={'/Books'}>Books</Link>
              </li>
              <li>
                <Link to={'/Favorite'}>Favorites</Link>
              </li>
            </ul>
          </nav>
          <Switch>
//Paths to access music home and books
          <Route path='/Home' component={Home}/>
            <Route path='/Music' component={Music}/>
            <Route path='/Books' component={Books}/>
            <Route path='/Favorite' component={Favorite}/>
          </Switch>
        </div>
      </Router>
    );
  }
}

export default App;
