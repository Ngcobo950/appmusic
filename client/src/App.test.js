import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import Favorite from './App';
import render from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("We pass the test!!",() => {
    const component = render.create(<Favorite/>)
    let app = component.toJSON();
    expect(app).toMatchSnapshot();
})
