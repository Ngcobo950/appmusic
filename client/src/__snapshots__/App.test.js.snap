*8*9-// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`We pass the test!! 1`] = `
Array [
  <div
    class="head"
  >
    <h1>
      APPLE SOUNDS & BOOKS STORE
    </h1>
  </div>,
  <div>
    <nav>
      <ul>
        <li>
          <a
            href="#/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="#/Music"
            onClick={[Function]}
          >
            Music
          </a>
        </li>
        <li>
          <a
            href="#/Books"
            onClick={[Function]}
          >
            Books
          </a>
        </li>
        <li>
          <a
            href="#/Favorite"
            onClick={[Function]}
          >
            Favorites
          </a>
        </li>
      </ul>
    </nav>
  </div>,
]
`;
